app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/representee/RepresenteeController.scala

```diff
@@ -222,7 +222,7 @@ class RepresenteeController @Inject() (
 
   def confirmPerson(): Action[AnyContent] =
     authenticatedActionWithSessionData.async { implicit request =>
-      withCapacitorOrPersonalRepresentativeAnswers(request) { (representativeType, journey, answers) =>
+      withCapacitorOrPersonalRepresentativeAnswers(request) { (_, journey, answers) =>
```

Comment:
Looks like the `WithCapacitorOrPersonalRepresentativeAnswers` needs to change to remove a function parameter. If so take it out there rather than applying the underscore

Reply:
No, that parameter is used on some calls in the file.